moveit_core/urdfdom_compatibility.h.in

```diff
@@ -0,0 +1,78 @@
+/*********************************************************************
```

Comment:
I disagree based on other instances of these ``.h.in`` files: [moveit_resources](https://github.com/ros-planning/moveit_resources/tree/master/include/moveit_resources), [moveit_core](https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_core/version), [geometric_shapes](https://github.com/ros-planning/geometric_shapes/blob/kinetic-devel/test/resources/config.h.in) - none have it in their root.

How about you put it in ``moveit_core/version`` since this is related to having the wrong version of urdfdom in Wily? I just don't like having a header file in the root of a package. Or if you don't like, a ``.cmake/`` folder?